system-parachains/asset-hubs/asset-hub-polkadot/src/weights/pallet_assets_pool.rs

```diff
@@ -47,26 +46,22 @@ use core::marker::PhantomData;
 /// Weight functions for `pallet_assets`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_assets::WeightInfo for WeightInfo<T> {
-	/// Storage: `PoolAssets::Asset` (r:1 w:1)
-	/// Proof: `PoolAssets::Asset` (`max_values`: None, `max_size`: Some(210), added: 2685, mode: `MaxEncodedLen`)
 	fn create() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `42`
-		//  Estimated: `3675`
-		// Minimum execution time: 10_327_000 picoseconds.
-		Weight::from_parts(10_767_000, 0)
-			.saturating_add(Weight::from_parts(0, 3675))
-			.saturating_add(T::DbWeight::get().reads(1))
-			.saturating_add(T::DbWeight::get().writes(1))
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 0_000 picoseconds.
+		Weight::from_parts(0, 0)
+			.saturating_add(Weight::from_parts(0, 0))
```

Comment:
@muharem why zero?

Reply:
@muharem  Because of this?
```
type CreateOrigin = NeverEnsureOrigin<AccountId>;
```
Shouldn't it better generate `Weight::MAX`?

Reply:
@muharem Because of this?
```
type CreateOrigin = NeverEnsureOrigin<AccountId>;
```
Is that setting ok?

Reply:
AHK has different setting:
```
type CreateOrigin =
		AsEnsureOriginWithArg<EnsureSignedBy<AssetConversionOrigin, sp_runtime::AccountId32>>;
```

Reply:
Yes, it's right. There is no really a private key to sign as `AssetConversionOrigin`. it does not make sense to me, no sure why done that way. But both ways are ok.

Reply:
ok, thank you